lib/server-impl.js

```diff
@@ -37,6 +38,7 @@ function createApp (options) {
 
 function start (opts) {
     const options = createOptions(opts);
+    process.env.TZ = options.timezone;
```

Comment:
It feels a  bit nasty to change an env variable runtime, any better ways to set the timezone for the node-process?